### DISCUSSION TITLE
Errors on import

### DISCUSSION THREAD

**Discussion by Waleeta** [Bunlong/react-papaparse#130] [General]
I am getting errors importing everything but `CSVReader` into my app:

<img width="917" alt="Screen Shot 2022-03-14 at 2 04 16 PM" src="https://user-images.githubusercontent.com/19413495/158255628-940fa944-0b81-4eaa-a8ca-9898d57f3588.png">
<img width="739" alt="Screen Shot 2022-03-14 at 3 27 04 PM" src="https://user-images.githubusercontent.com/19413495/158255639-06119b7a-625c-4fe0-adfa-1e90add8f8f4.png">
<img width="812" alt="Screen Shot 2022-03-14 at 3 29 31 PM" src="https://user-images.githubusercontent.com/19413495/158255960-975a3363-0a29-41cb-9c05-c6a30ce9e9dc.png">

My node version is `^17.0`. Any idea why these won't import? I'm not sure why CSVReader will and nothing else.

**Comment by Bunlong** [comment]
@Waleeta Which react-papaparse version are you using? Could you share us your code?

**Reply by Waleeta** [reply]
Yes - I updated to your latest version (I was at ~1.something, it was very old), and that fixed these errors - I converted and imported all the `useCSVReader` where necessary, but now my error is: 

<img width="971" alt="Screen Shot 2022-03-15 at 9 26 49 AM" src="https://user-images.githubusercontent.com/19413495/158399993-0eb25f6a-45d2-4d1e-9121-0e76dd40a87f.png">
<img width="933" alt="Screen Shot 2022-03-15 at 9 27 03 AM" src="https://user-images.githubusercontent.com/19413495/158400001-112269d5-0bfe-490d-a19f-07102abf9992.png">

**Reply by Waleeta** [reply]
I uninstalled and reinstalled everything, this error was coming from a particular call to `<CSVReader>`, still unclear why but all others are working.
